src/Raven.Server/Config/Categories/SecurityConfiguration.cs

```diff
@@ -143,8 +143,15 @@ public class SecurityConfiguration : ConfigurationCategory
         [DefaultValue(null)]
         [ConfigurationEntry("Security.WellKnownCertificates.Admin", ConfigurationEntryScope.ServerWideOnly)]
         public string[] WellKnownAdminCertificates { get; set; }
+        
+        
+        [Description("Well known issuer certificate in base 64 format or a file path that will be used to validate a new client certificate when the issuer's certificate has changed.")]
+        [DefaultValue(null)]
+        [ConfigurationEntry("Security.WellKnownIssuers.Admin", ConfigurationEntryScope.ServerWideOnly)]
+        public string[] WellKnownIssuers { get; set; }
```

Comment:
Do we put the whole cert here? Why not thumbprints list?

Reply:
We put either the whole certificate as base64 or a path to the cert, yes.
The problem is that you need the full cert to figure out who the issuer is.

We explicitly error if the issuer certificate has the private key, which will ensure that this is never there.

Reply:
And if you want to pass multiple, you'd use `;` right?